pynuodb/datatype.py

```diff
@@ -158,27 +158,29 @@ def __cmp__(self, other):
            "string": STRING,
            "char": STRING,
            "varchar": STRING,
-           "text": STRING,
```

Comment:
Why is this line removed?

TEXT is an alias for CLOB, so perhaps this should be BINARY like "clob" below.

Reply:
The server will never return this type.  You can check the Types/Type.cpp file to see the possible returned names for various SQL types.